tools/templates/moar/Makefile.in

```diff
@@ -14,8 +14,7 @@ MOAR_PREFIX = @nfp(@moar::prefix@)@
 # MOAR and M_NQP are only used in command lines, ok to quote
 MOAR   = @nfpq(@moar::bindir@/moar@moar::exe@)@
 M_NQP  = @nfpq(@m_nqp@)@
-M_LIBDEFPATH = $(PREFIX)@nfp(/share/nqp/lib)@
-M_LIBPATH = $(LIBDIR)@nfp(/nqp/lib)@
+M_LIBDIR = $(PERL6_HOME)@nfp(/lib)@
```

Comment:
`@nfp($(PERL6_HOME)/lib)@` would make it more readable.

Reply:
Fixed.